### ISSUE TITLE
Some tables do not fit with the data-out-DB approach

### ISSUE THREAD

**Issue by matamadio** [GFDRR/rdl-data#36]
The tables:

1. hazard.footprint_data
2. ged4all.cost, occupants and tags
3. loss.map_values
4. loss.curve_map_values

are meant to store data, consistently with the data-in-DB approach, and are substituted with link to S3 file in the current approach.
- 1 can be removed safely, as it only stores hazard value;
- 3 would store loss in relation to asset; in the most common case of vector or table ADM map with losses per unit that can be part of the file table itself, ane be removed from metadata attributes. 
- 4 is similar to 3, here we also have the "rates" field which is the specification associated to loss value (could be simply name of loss field in the table, eg rate_loss)
- 2 is problematic because there are 3 tables that could link to individual geometries (e.g vector buildings each with different cost, occupants and tags) in one exposure dataset; this is a bit trickier to untangle.

**Comment by matamadio** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Review schema and optimisation](https://trello.com/c/F3LRwVcr/42-review-schema-and-optimisation)

**Comment by stufraser1** [comment]
This is a question on which I'd like to hear the views of Paul Henshaw at GEM. He may be able to comment on the implications of these proposals in relation to original aims and subsequent decisions that led to this structure. We should note that this proposal arises because of issues found when implementing the DB i.e. scaling database size and serving data to users with the data contained in the database, which were not necessarily experienced in the pilot/development phase

**Comment by matamadio** [comment]
Yes, the exposure schema is the most impacted by the practical separattion of data and metadata, because 3 tables refer to individual objects in dataset (differently from other schemas). Same reason why it doesn't go along at best with aggregated exposure data. Ultimately, I found it less flexible than H and L schema.

**Comment by matamadio** [comment]
Tables cut in MVP.
Keep issue open for long-term consideration.

**Comment by matamadio** [comment]
Issue still relevant for GEM-compliant exposure schema, expecting to have tables describing in-dataset range of attributes.
